app/experimenter/experiments/bugzilla.py

```diff
@@ -164,4 +164,7 @@ def create_experiment_bug(experiment):
 
 def get_bugzilla_id(bug_url):
     query = urlparse(bug_url).query
-    return int(parse_qs(query)["id"][0])
+    try:
+        return int(parse_qs(query)["id"][0])
```

Comment:
Do this exact check but in the form at validation time so we don't even let them enter it

Reply:
Oh but we also have to do this check here as well for the case that someone already entered it in some invalid form in the production database by teh time we launch this code, but if you can't parse the id out of the url qs then just fail silently and pass as if the url wasn't set at all.